### PR TITLE
update OWNERS.md and add release guide in README.md

### PR DESCRIPTION
**Pull request by neolit123**
first commit is an OWNERS update:
- @KentaTada @odinuge as reviewers. thanks for your contributions!
- @SataQiu as approver

remove approvers to sync with https://github.com/kubernetes/kubernetes/pull/98547
- @luxas, @timothysc, @rosti, @ereslibre 

please state if you object to these changes.
applying lazy consensus next week ~4th of Feb.

@KentaTada i think you are not a member of the kubernetes github org?
if so, i'd be happy to sponsor your membership. likely @odinuge can +1 as well.

https://github.com/kubernetes/community/blob/master/community-membership.md#requirements
it requires sending a PR to k/org.

second commit adds a small release guide to README.md.

### CODE REVIEW

**Comment by KentaTada** [comment]
>@KentaTada i think you are not a member of the kubernetes github org?
>if so, i'd be happy to sponsor your membership. likely @odinuge can +1 as well.

I  am not a member of the kubernetes github org.
@neolit123 @odinuge  Could you sponsor my membership?
https://github.com/kubernetes/org/issues/2467

Thanks!

**Review by odinuge — APPROVED** [review]
Thanks!

> @KentaTada i think you are not a member of the kubernetes github org?
> if so, i'd be happy to sponsor your membership. likely @odinuge can +1 as well.

Yes, I do! Already commented on your k8s/org issue. :rocket: 

/lgtm

**Comment by neolit123** [comment]
ok, once https://github.com/kubernetes/org/issues/2467 is closed we can proceed here. :+1:

**Comment by neolit123** [comment]
> kubernetes/org#2467

merged, looking for LGTM again.
instead of removing the old approvers moved them to "emeritus_approvers"

**Comment by odinuge** [comment]
/lgtm

Not sure if you want me to put it into tide now... But feel free to cancel hold if that was what you wanted! :tada: 
/hold

**Comment by KentaTada** [comment]
Thank you for all your help.
/hold cancel

**Pull request merged**
